### ISSUE TITLE
b-form-datepicker calendar button does not meet WCAG

### ISSUE THREAD

**Issue by jhaagen** [bootstrap-vue/bootstrap-vue#7217]
### Describe the bug

The calendar SVG in the `<b-form-datepicker>` is wrapped by a button element, this button element needs to have a ARIA attribute. This currently fails accessibility requirements from multiple sources (Lighthouse, SiteImprove, etc.)

### Steps to reproduce the bug

1. View page with `<b-form-datepicker>` element in Chrome
2. Run Lighthouse against that page
3. View accessibility results
4. See the error `Buttons do not have an accessible name`

### Expected behavior

The button needs to have an ARIA label attached to it instead of just the SVG inside of the button to be compliant

### Versions

**Libraries:**

- BootstrapVue: 2.5.0
- Bootstrap: 4.#.#
- Vue: 2.#.#

**Environment:**

- Device: Any
- OS: Any
- Browser: Chrome
- Version: Any

### Demo link

This is visible on the documentation page for this element on the Bootstrap Vue site: https://bootstrap-vue.org/docs/components/form-datepicker

### Additional context

Add any other context about the bug here.
